frontend/ui/test/widgets/create_trip_widget_test.dart

```diff
@@ -33,26 +62,38 @@ void main() {
     when(tripService.createTrip(any))
         .thenAnswer((t) => Future.value(t.positionalArguments[0]));
 
-    var createTripsWidget = CreateTripWidget(tripService);
+    var createTripsWidget = CreateTripWidget(tripService, placesApiServices);
     await tester.pumpWidget(wrapForDirectionality(createTripsWidget));
     await tester.pumpAndSettle();
 
-    // TODO: Consider enabling this after fixing the test/code. The display
-    // behavior has changed since this was first introduced.
+    await tester.enterText(
+        find.widgetWithText(TextField, 'enter your trip name'), 'London Trip');
```

Comment:
Is this supposed to be lower case? And if it is could you fix it in the widget and test?